public/paths/hubs/roles/role.yml

```diff
@@ -0,0 +1,120 @@
+get:
+  operationId: "getRole"
+  tags:
+    - Role
+  parameters:
+    - name: roleId
+      description: The ID of the Role.
+      in: path
+      required: true
+      schema:
+        type: string
+  summary: Fetch Role
+  description: Requries the `hubs-roles-manage` capability.
```

Comment:
look at my open branch with some of the updates to these endpoints, and follow my pattern there please.